spec/robots/dor_repo/caption/end_caption_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Robots::DorRepo::Caption::EndCaption do
+  let(:druid) { 'druid:bb222cc3333' }
+  let(:robot) { described_class.new }
+
+  let(:object) { build(:dro, id: druid) }
+  let(:workspace_client) { instance_double(Dor::Services::Client::Workspace, create: nil) }
```

Comment:
I noticed that `create: nil` isn't needed here, I guess because `create()` doesn't actually get called? If we don't expect it to be called I suggest we remove this since it implies that it will.